Index: include/guichan/key.hpp
===================================================================
--- include/guichan/key.hpp	(wersja 1055)
+++ include/guichan/key.hpp	(kopia robocza)
@@ -118,7 +118,7 @@
             SPACE              = ' ',
             TAB                = '\t',
             ENTER              = '\n',
-            LEFT_ALT           = 1000,
+            LEFT_ALT           = -1000,
             RIGHT_ALT,
             LEFT_SHIFT,
             RIGHT_SHIFT,
